Refactored servo tab

egonl opened the pull request:
Refactored servo tab for RF2

![image](https://user-images.githubusercontent.com/34315684/215609472-26d5cb59-ea99-442e-abd1-5f24ead86ffb.png)

egonl commented:
![image](https://user-images.githubusercontent.com/34315684/215609472-26d5cb59-ea99-442e-abd1-5f24ead86ffb.png)

pkaig commented:
The tab looks good Egon. Only suggestions.
* Maybe it needs the 'Unusual values warning' if not all (S1-S3) 'Geometry correction' selected Or JS to select all 3 if any are selected.
* Does it need 'save' and 'save and reboot'? It may cause confusion. when the wrong 'save' is used. Could either show 'save and reboot' for all or select either depending on the requirements for that specific parameter.
* Remove servo configuration help icon on top bar. It just says the same as the title and you have individual icons for each parameter so it isn't needed.

egonl commented:
Thank Phil! I implemented:
- A warning if the geometry correction is not equal for the first 2 (FP) or 3 servos.
- Removed the redundant tooltip and redundant note.
- It now only shows the 'Save and reboot' button if you change a rate. That's because rates can only be set at boot time. All the other changes take effect immediately.
